src/include/symbol_table/symbol_table.h

```diff
@@ -0,0 +1,138 @@
+#ifndef JUCC_SYMBOL_TABLE_H
+#define JUCC_SYMBOL_TABLE_H
+
+#include <string>
+#include <unordered_map>
+#include <utility>
+#include <vector>
+
+namespace jucc {
+
+namespace symbol_table {
+
+struct Node {
+  /**
```

Comment:
please use class, please

Reply:
No struct is better for this purpose. I insist it staying as a struct.

Reply:
I'm just asking for uniformity throughout the codebase. Both serve the same purpose.
Rest it's up to you ;)

Reply:
🍿

Reply:
please fight

Reply:
please join

Reply:
NO